src/components/ConnexionLogin.jsx

```diff
@@ -23,7 +23,12 @@ export default function useForm({ setToken }) {
     api.post("/auth/login", values).then((res) => {
       setToken(res.data.token);
       dispatch({ type: "SET_ID", test: res.data.idClient });
-      history.push("/clientpage");
+      dispatch({ type: "SET_TOKEN", test: res.data.token });
+      if ((res.data.idClient = 1)) {
```

Comment:
- Don't use specific, arbitrary values like `1`
- I think you wanted to test `res.data.idClient === 1`